rfcs/20201113-two_photon_lithography-1.2.0.md

```diff
@@ -0,0 +1,116 @@
+	# nanoMFG Software Planning Document
+<!-- Replace text below with long title of project:short-name -->
+## nanoMFG Tool for Two-Photon Lithography (TPL)
+### Target Release: v1.2.0: Dec 31st, 2020
+
+## Development Team
+<!-- Complete table for all team members 
+ roles: PI, developer, validation
+ status: active, inactive
+-->
+Name | Role | github user | nanohub user | email | status
```

Comment:
@singhra2 Can you indicate if any of the listed developers should also be reviewers by updating their role to "developer/reviewer".  Also add any addition team members/stakeholders from the broader TPL effort to the project team as either reviewrs or contributors and I will add them to the review team.